openmmtools/alchemy.py

```diff
@@ -1287,7 +1606,140 @@ def _alchemically_modify_HarmonicBondForce(reference_force, alchemical_region):
                 # Standard bond.
                 force.addBond(particle1, particle2, theta0, K)
 
-        return [force, custom_force]
+        return {'': [force], 'lambda_bonds': [custom_force]}
+
+    def _get_electrostatics_energy_expressions(self, reference_force):
```

Comment:
What happens if someone specifies an exact force and not PME?

Reply:
As long as `reference_force` is a `NonbondedForce`, the method should behave correctly. We call this only inside `_alchemically_modify_NonbondedForce`.